packages/elements/src/datetime-picker/__test__/datetime-picker.value.test.js

```diff
@@ -9,108 +9,78 @@ describe('datetime-picker/Value', () => {
+      expect(calendarEl.max).to.be.equal('2020-04-30', 'calendar max getter is wrong');
     });
     it('It should not be possible to set from value after to', async () => {
       const el = await fixture('<ef-datetime-picker lang="en-gb" range values="2020-04-30,2020-04-01"></ef-datetime-picker>');
-      expect(el.error).to.be.equal(true);
+      expect(el.checkValidity()).to.be.equal(false, 'from value is after to');
     });
     it('It should not be possible to set value before min', async () => {
       const el = await fixture('<ef-datetime-picker lang="en-gb" min="2020-04-22" value="2020-04-21"></ef-datetime-picker>');
-      expect(el.error).to.be.equal(true);
+      expect(el.checkValidity()).to.be.equal(false, 'value is less than min');
     });
     it('It should not be possible to set value after max', async () => {
       const el = await fixture('<ef-datetime-picker lang="en-gb" max="2020-04-20" value="2020-04-21"></ef-datetime-picker>');
-      expect(el.error).to.be.equal(true);
-    });
-    it('While typing the value calendar input should not randomly update value', async function () {
-      if (isIE()) {
-        this.skip();
-      }
-      // this test becomes invalid if date-fns ever supports strict formatting
-      const el = await fixture('<ef-datetime-picker lang="en-gb" opened></ef-datetime-picker>');
-      const input = el.inputEl;
-      await triggerFocusFor(input);
-      typeText(el.inputEl, '21-A-2020');
-      await elementUpdated(el);
-      expect(el.inputEl.value).to.be.equal('21-A-2020', 'While in focus input value is not changed');
-      await triggerFocusFor(el);
-      await elementUpdated(el);
-      expect(el.inputEl.value).to.be.equal('21-Apr-2020', 'On blur input values becomes formatted value');
+      expect(el.checkValidity()).to.be.equal(false, 'value is more than max');
     });
     it('It should be possible to select value by clicking on calendar', async () => {
       const el = await fixture('<ef-datetime-picker lang="en-gb" opened view="2020-04"></ef-datetime-picker>');
-      const calendarEl = el.calendarEl;
+      const calendarEl = calendarElement(el);
       await elementUpdated(el);
       const cell = calendarEl.shadowRoot.querySelectorAll('div[tabindex]')[2]; // 2020-04-01
       cell.click();
       await elementUpdated(el);
       expect(el.value).to.be.equal('2020-04-01', 'Value has not update');
-      expect(el.inputEl.value).to.be.equal('01-Apr-2020', 'Input value has not updated');
+      expect(inputElement(el).value).to.be.equal('2020-04-01', 'Input value has not updated');
     });
     it('It should be possible to select value in range duplex mode', async () => {
       const el = await fixture('<ef-datetime-picker lang="en-gb" opened range duplex></ef-datetime-picker>');
       el.views = ['2020-04', '2020-05'];
       await elementUpdated(el);
-      await nextFrame();
-      await nextFrame();
+      await nextFrame(el);
+      await nextFrame(el);
```

Comment:
Here and in other places. I think these multiple `nextFrame` calls were added to accommodate IE11. Please double check if you still need them.

Reply:
Updated. It still need to call nextFrame one time after updating the views.